tests/test_scripts.py

```diff
@@ -1540,6 +1540,43 @@ def test_split_paired_reads_2_fq():
     assert n > 0
 
 
+def test_split_paired_reads_3_output_dir():
+    # test input file
+    infile = utils.get_test_data('paired.fq')
+
+    ex_outfile1 = utils.get_test_data('paired.fq.1')
+    ex_outfile2 = utils.get_test_data('paired.fq.2')
+
+    # actual output files...
+    outfile1 = utils.get_temp_filename('paired.fq.1')
+    in_dir = os.path.dirname(outfile1)
+    outfile2 = utils.get_temp_filename('paired.fq.2', in_dir)
+
+    script = scriptpath('split-paired-reads.py')
+    args = [infile, '--output-dir', in_dir]
+
+    utils.runscript(script, args, in_dir)
```

Comment:
This will work without specifying in_dir, and, more importantly, specifying in_dir means that this test works without specifying --output-dir, because run_script changes to in_dir in this case!